vector/build.gradle

```diff
@@ -307,14 +264,12 @@ dependencies {
     implementation 'commons-codec:commons-codec:1.15'
 
     // MapTiler
-    fdroidApi(libs.maplibre.androidSdk) {
+    api(libs.maplibre.androidSdk) {
         exclude group: 'com.google.android.gms', module: 'play-services-location'
     }
-    fdroidApi(libs.maplibre.pluginAnnotation) {
+    api(libs.maplibre.pluginAnnotation) {
```

Comment:
Will we have a pb here for `gplay`? group exclusion was not active for this variant. Just to double check.

Reply:
it's excluded from the `mapLibre` dependency and then added back at application level for the _gPlay_ variant https://github.com/vector-im/element-android/pull/6788/files#diff-6998d5441936ca2baa15113a1f9f709549aeaaf929fedc553b94a67cc9a94833R374